### PR TITLE
feat(js): Adds further required support for context vars

### PR DESCRIPTION
**Pull request by jacoblee93**

### CODE REVIEW

File: js/src/run_trees.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export function convertToDottedOrderFormat(
   );
 }
 
+export const _LC_CONTEXT_VARIABLES_KEY = Symbol.for("lc:context_variables");
```
Comment (dqbd) [inline js/src/run_trees.ts:31]: _nit_: do we want to export this symbol out? Ideally would avoid exporting these internals out. If no-option is available, maybe `@hidden` jsdoc tag would help (saw that tag somewhere when reading LC.js)

**Review by dqbd — APPROVED** [review]

File: js/src/run_trees.ts (hunk 1 above)
Comment (jacoblee93) [reply js/src/run_trees.ts:31]: Yeah can refactor

Comment (jacoblee93) [reply js/src/run_trees.ts]: Done

**Comment by jacoblee93** [comment]
Thanks!

**Pull request merged**